py/strato/common/log/log2text.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, relativeTime, withThreads, showFullPaths, noDebug, microsecon
         self._firstClock = None
         self._clock = self._relativeClock if relativeTime else self._absoluteClock
         self._relativeClockFormat = "%.6f" if microsecondPrecision else "%.3f"
-        self._minimumLevel = logging.INFO if noDebug else logging.DEBUG
+        self._minimumLevel = self._min_level(min_level, noDebug)
```

Comment:
why would you pass both of the parameters and not decide on min-level before the init? think that would be cleaner